src/mono/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.Mono.cs

```diff
@@ -8,6 +8,45 @@ namespace System.Runtime.InteropServices
 {
     public static unsafe partial class MemoryMarshal
     {
+
+        /// <summary>
```

Comment:
If there is some special tweak needed for Mono, just add ifdef into the shared implementation instead of duplicating it 3x.